docs/index.md

```diff
@@ -22,6 +23,7 @@ Is something broken?
 * [Who's OnCall? (PagerDuty schedules)](https://dsva.pagerduty.com/schedules)
 * [See OnCall documentation](https://github.com/department-of-veterans-affairs/vets.gov-team/tree/master/Practice%20Areas/Engineering/OnCall) (internal)
 * [Breakers dashboard (Grafana)](http://grafana.vetsgov-internal/dashboard/db/breakers?orgId=1)
+* Submit an Issue to Triage [HERE](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=triage%2C+triage-incident&template=triage-incident-template.md&title=)
```

Comment:
```suggestion
* Submit a non-critical issue to be triaged [HERE](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=triage%2C+triage-incident&template=triage-incident-template.md&title=)
```

couple wording changes, is it fair to imply that filing an issue should be for less critical things? (i.e. real production issues should be handled in #oncall or with paging)